common/src/test/scala/com/gu/stepfunctions/StepFunctionsSpec.scala

```diff
@@ -5,9 +5,10 @@ import com.amazonaws.services.stepfunctions.AWSStepFunctionsAsyncClientBuilder
 import com.gu.aws.CredentialsProvider
 import com.gu.test.tags.annotations.IntegrationTest
 import com.typesafe.scalalogging.LazyLogging
-import org.scalatest.{AsyncFlatSpec, Matchers}
+import org.scalatest.{AsyncFlatSpec, Ignore, Matchers}
 
 @IntegrationTest
+@Ignore //ignore this by default because it takes a very long time to run
```

Comment:
Are we running this test in team city?

Reply:
No, we don't run any integration tests in TC because it doesn't have the necessary credentials.